win32ss/printing/include/marshalling/printerdrivers.h

```diff
@@ -70,6 +70,7 @@ static const MARSHALLING PrinterDriver5Marshalling = {
     }
 };
 
+/* TODO: Add PrinterDriver6Marshalling and PrinterDriver8Marshalling */
```

Comment:
I'm fine with accepting TODOs for stuff that is supported in NT5.x.
However, `DRIVER_INFO_8` was only added in NT6 according to MSDN.

On the other hand, there are structures like `JOB_INFO_4`, which also exist in NT5.x, but don't exist in our printing implementation for whatever reason and also not in your TODO comments.
If you already add TODOs for unimplemented printing structures, please check that you haven't missed any NT5.x one.